### PR TITLE
feat: add kf forum banner + link in the app menus

### PR DESCRIPTION
**Pull request by evans-g-crsj**
screenshot_forum_banner

TODO: link ref is missing!

### CODE REVIEW

File: src/components/Header/AppsMenu.js [hunk 1]
```diff
@@ -28,6 +29,17 @@ const AppsMenu = () => {
         Studies and Access
       </a>
     </Menu.Item>,
+    <Menu.Item key="kf-forum">
+      <a href={`${'todo'}/todo`} target={'_blank'} rel={'noopener noreferrer'}>
+        <MessageOutlined style={{ fontSize: '15px', marginRight: 0 }} />
```
Comment (evans-g-crsj) [inline src/components/Header/AppsMenu.js:34]: it seems that we're forced to use inline-styling

Comment (francisl) [reply src/components/Header/AppsMenu.js:34]: why? https://ant.design/components/icon/#API has an api for className.

File: src/ForumBanner.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+.forum-link {
+  text-decoration: underline;
+}
+
+.new-badge {
+  margin-right: 8px;
+  margin-left: 12px;
+}
```
Comment (francisl) [inline src/ForumBanner.scss:14]: Please use sass module if you can. Especially if you want to use short names.

File: src/ForumBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+const SHOW_FORUM_BANNER = 'SHOW_FORUM_BANNER';
+
+export const removeForumBanner = () => localStorage.removeItem(SHOW_FORUM_BANNER);
+export const showForumBanner = () => localStorage.getItem(SHOW_FORUM_BANNER) === 'true';
+export const setForumBannerForDisplay = () => localStorage.setItem(SHOW_FORUM_BANNER, 'true');
+
+const ForumBanner: FunctionComponent = () => (
+  <Alert
+    message={
+      <div className={'message-container'}>
```
Comment (francisl) [inline src/ForumBanner.tsx:14]: see above, css module would be better here. plus you don't need the `{''}`

File: src/components/Header/AppsMenu.js [hunk 4]
```diff
@@ -28,6 +29,17 @@ const AppsMenu = () => {
...
+        Kids First Forum{' '}
+        <Badge
+          count={'New'}
```
Comment (francisl) [inline src/components/Header/AppsMenu.js:37]: no need to add extra parsing here `{''}` =>  `=''`

File: src/components/Header/AppsMenu.js [hunk 5]
```diff
@@ -49,7 +61,14 @@ const AppsMenu = () => {
   return (
     <HeaderMenu menuItems={menuItems}>
       <AllAppsMenuIcon size="14px" />
-      Resources
+      <Badge
+        count={'New'}
+        className={'resourcesBadge'}
+        style={{ backgroundColor: '#52c41a' }}
```
Comment (francisl) [inline src/components/Header/AppsMenu.js:67]: in css

**Review by francisl — CHANGES_REQUESTED** [review]

**Review by francisl — APPROVED** [review]

**Pull request merged**